pkg/securitycontext/util.go

```diff
@@ -47,6 +47,14 @@ func HasCapabilitiesRequest(container *api.Container) bool {
 	return len(container.SecurityContext.Capabilities.Add) > 0 || len(container.SecurityContext.Capabilities.Drop) > 0
 }
 
+// HasSELinuxOptions returns true if the containers security context specifies SELinux options
+func HasSELinuxOptions(container *api.Container) bool {
```

Comment:
Remember that there will soon be a pod-level selinux context -- you might want to pass the pod, too.